src/_includes/docs/install/compiler/xcode.md

```diff
@@ -82,9 +82,9 @@ With Xcode, you can run Flutter apps on an iOS device or on the simulator.
 
 {% endif %}
 
-{% if time=="first" %}
+{% if attempt=="first" %}
 
-### Install CocoaPods
+### Install CocoaPods (Optional)
```

Comment:
Optional is misleading. It's required if you use any package with native deps on iOS or macOS. I'd argue it's required for non trivial development on iOS